ingest-app/src/cmr/ingest/api/routes.clj

```diff
@@ -110,16 +111,24 @@
           (GET "/status/:task-id"
                [task-id :as request]
                (bulk/get-provider-task-status provider-id task-id request)))))
-    ;; Variable ingest routes
+    ;; Variables ingest routes
     (context "/variables" []
       (POST "/"
             {:keys [request-context headers body]}
             (variables/create-variable request-context headers body))
-      (context "/:variable-key" [variable-key]
-        (PUT "/"
-             {:keys [request-context headers body]}
-             (variables/update-variable
-              request-context headers body variable-key))))))
+      (PUT "/:variable-id"
+           [variable-id :as {:keys [request-context headers body]}]
+           (variables/update-variable
+            request-context headers body variable-id)))
+    ;; Services ingest routes
+    (context "/services" []
+      (POST "/"
```

Comment:
On second thought, since we are generating the `native-id` for Variables and Services during create, they are different from Collections and Granules, I am OK with the way it is now.